src/hyperloglog.c

```diff
@@ -227,6 +231,19 @@ static int simd_enabled = 1;
 #define HLL_USE_AVX2 0
 #endif
 
+#ifdef __aarch64__
+static int simd_enabled = 1;
+#define HLL_USE_NEON (simd_enabled)
+#else
+#define HLL_USE_NEON 0
+#endif
+
+#if defined(HAVE_X86_SIMD) || defined(__aarch64__)
```

Comment:
HAVE_X86_SIMD is always defined, either to 1 or to 0.

```suggestion
#if HAVE_X86_SIMD || defined(__aarch64__)
```